src/Microsoft.Azure.WebJobs.Host/InvocationFilterAttribute.cs

```diff
@@ -0,0 +1,39 @@
+﻿// InvocationFilterAttribute.cs
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// This is the first iteration for the Invocation Filter Attribute
```

Comment:
Put the proper code comments in place now. E.g. "Defines an interface for declarative invocation filters."